### ISSUE TITLE
OpenProject fails on non-empty namespace

### ISSUE THREAD

**Issue by mkromberg** [aplteam/Cider#11]
If you try to create a project from a namespace which already contains some code, CreateProject fails at the very last step. It should instead write the contents of the namespace to source files, IMHO.
```
      ]CIDER.CreateProject c:\tmp\myproj myapp                                                                            
"c:/tmp/myproj" does not exist yet - create? (Y/n) y
Project successfully created; open as well? (Y/n) y 
 Loading project from c:\tmp\myproj/ 
 Checking parameters... 
   All fine 
 Bringing in the source code... 
Target namespace "#.myapp" is not empty. Delete contents? (y/n) n
* Command Execution Failed: Cannot Link into non-empty namespace
```

**Comment by aplteam** [comment]
Yes, it should. 

Going to be fixed in 0.15.0

**Comment by aplteam** [comment]
Done
